src/In.ProjectEKA.HipService/DataFlow/DataFlowArtefactRepository.cs

```diff
@@ -0,0 +1,23 @@
+
+namespace In.ProjectEKA.HipService.DataFlow
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using In.ProjectEKA.HipLibrary.Patient.Model;
+    
+    public class DataFlowArtefactRepository : IDataFlowArtefactRepository
```

Comment:
It could have been simpler by just sending consentArtefactID returns Artefact, and caller should use artefact to prepare the dataflowartefact?

Reply:
Once Implementing that story, I will fix it as I will have a better context then.

Reply:
which story?